test/UnitTest/v2/v2data.py

```diff
@@ -495,3 +501,4 @@
             ],
             "updateAction": "UPDATE"
 }
+
```

Comment:
Speeling correction required in createDataWithupdateaction{}
=>   "updateAction": "CRETAE"